cobra/__init__.py

```diff
@@ -194,11 +195,11 @@ def __init__(self, socket, sflags=0):
                 raise Exception('Missing "msgpack" python module ( http://visi.kenshoto.com/viki/Msgpack )')
 
             def msgpackloads(b):
-                logger.debug("<< %r  (%r)", b, loadargs)
+                logger.log(e_cmn.MIRE, "<< %r  (%r)", b, loadargs)
```

Comment:
i love these getting logged, but they belong at a deeper level
same for `msgpackdumps`

Reply:
as i mentioned in the other comment, this back and forth has saved so very much troubleshooting time, and was vital for solving some critical bugs a long time ago.
because of the way Python does logging, and because we're using the correct syntax, this should have minimal impact unless you choose to be at MIRE log level.